uiowa/uiowa
# Possible fix to resolve mb_strtolower() error

Resolves https://github.com/uiowa/uiowa/issues/6211.  

# How to test

- Pull down https://sandbox.uiowa.ddev.site/bannerariadescribedby-test-page on main and view the error message while logged in. 
- Switch to this pr and clear the cache and verify that error message no longer displays while the `banner_aria_describedby` is still working if there is a `banner_title`.
